.github/workflows/cassandra.yml

```diff
@@ -40,3 +40,19 @@ jobs:
       run: docker compose -f test/cluster/cassandra/docker-compose.yml logs
     - name: Remove cluster
       run: docker compose -f test/cluster/cassandra/docker-compose.yml down
+
+    # TODO: delete the below and move RUSTFLAGS into the above test run when cassandra 5.0.0 releases.
+    - name: Setup 3-node Cassandra cluster
+      run: |
+        docker compose -f test/cluster/cassandra5/docker-compose.yml up -d --wait
+    - name: Run tests on cassandra 5 beta
+      run: |
+        CDC='disabled' RUSTFLAGS="--cfg cassandra_tests" RUST_LOG=trace SCYLLA_URI=172.42.0.2:9042 SCYLLA_URI2=172.42.0.3:9042 SCYLLA_URI3=172.42.0.4:9042 cargo test --verbose --features "full-serialization" -- --skip test_views_in_schema_info --skip test_large_batch_statements
+    - name: Stop the cluster
+      if: ${{ always() }}
+      run: docker compose -f test/cluster/cassandra5/docker-compose.yml stop
+    - name: Print the cluster logs
+      if: ${{ always() }}
+      run: docker compose -f test/cluster/cassandra5/docker-compose.yml logs
+    - name: Remove cluster
+      run: docker compose -f test/cluster/cassandra5/docker-compose.yml down
```

Comment:
This will be done sequentially - first we perform tests on normal cassandra, then on beta.
Cassandra tests are already longest part of our CI. Maybe it would be better to create a separate workflow for this?

Reply:
I have moved it into a separate workflow as requested.